### ISSUE TITLE
FutureWarning .as_matrix() > .values

### ISSUE THREAD

**Issue by sablanchard** [UDST/pandana#103]
Getting a future warning, for example:
```
FutureWarning: Method .as_matrix will be removed in a future version. Use .values instead.
  distances, indexes = self.kdtree.query(xys.as_matrix())
```

should be updated next release.

**Comment by smmaurer** [comment]
Update: 

This has been partially addressed, but it looks like there are still instances of `.as_matrix` in `network.py`, `test_cyaccess.py`, and `tmp.py`.

**Comment by smmaurer** [comment]
Resolved in PR #124.
